<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Config_Widget</class>
 <widget class="QWidget" name="Config_Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>494</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>360</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>480</width>
    <height>380</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Настройки ПИД</string>
  </property>
  <widget class="QLabel" name="label_info_for_user">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Пропорциональный сигнал по высоте:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2_info_for_user">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Дифференциальный сигнал по высоте:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3_info_for_user">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>271</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Пропорциональный сигнал поворота через крен:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4_info_for_user">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>261</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Дифференциальный сигнал поврота через крен:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5_info_for_user">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>261</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Пропорциональный сигнал для тонгажа:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6_info_for_user">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>251</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Пропорциональный сигнал для крена:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7_info_for_user">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Пропорциональный сигнал для рысканья:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8_info_for_user">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Дифференциальный сигнал для тонгожа:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9_info_for_user">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Дифференциальный сигнал для крена:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10_info_for_user">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>251</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Дифференциальный сигнал для рысканья:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Get_PID">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>460</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Загрузить ПИД</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Set_PID">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>460</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Отгрузить ПИД</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Close">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>460</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Закрыть </string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_P_Alt">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>10</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <double>99.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_2_D_Alt">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>40</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <double>99.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_3_P_Bank">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>70</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <double>10.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_4_D_Bank">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>100</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <double>10.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_5_P_Pitch">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>130</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <double>10.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_6_P_Roll">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>160</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <double>10.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_7_P_Yaw">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>190</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <double>10.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_8_D_Pitch">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>220</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <double>10.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_9_D_Roll">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>250</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <double>10.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_10_D_Yaw">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>280</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <double>10.000000000000000</double>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_del_pid">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>460</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Очистить ПИД</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11_info_for_user">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>10</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Максимальный вывод для тонгажа:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12_info_for_user">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>40</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Миниммальный вывод для тонгожа:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13_info_for_user">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>70</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Максимальный вывод для крена:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14_info_for_user">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>100</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Минимальный вывод для крена:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15_info_for_user">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>130</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Максимальный вывод для рысканья:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_16_info_for_user">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>160</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Минимальный вывод для рысканья:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_17_info_for_user">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>10</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Тример левого рудеватора:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_18_info_for_user">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>40</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Тример правого рудеватора: </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_19_info_for_user">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>70</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Тример левого элерона:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_20_info_for_user">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>100</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Тример правого элерона:</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_Max_pitch">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>10</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-90.000000000000000</double>
   </property>
   <property name="maximum">
    <double>90.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_Min_pitch">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>40</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-90.000000000000000</double>
   </property>
   <property name="maximum">
    <double>90.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_Max_roll">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>70</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-90.000000000000000</double>
   </property>
   <property name="maximum">
    <double>90.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_Min_roll">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>100</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-90.000000000000000</double>
   </property>
   <property name="maximum">
    <double>90.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_Max_yaw">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>130</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-90.000000000000000</double>
   </property>
   <property name="maximum">
    <double>90.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_Min_yaw">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>160</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-90.000000000000000</double>
   </property>
   <property name="maximum">
    <double>90.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_trimm_left_Ruder">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>10</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <double>180.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_trimm_right_Ruder">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>40</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <double>180.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_trimm_left_Eleron">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>70</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <double>180.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_trimm_right_Eleron">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>100</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <double>180.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_21_info_for_user">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>190</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Максимальный вывод бочки:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_22_info_for_user">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>220</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Минимальный вывод бочки:</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_Max_bank">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>190</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-90.000000000000000</double>
   </property>
   <property name="maximum">
    <double>90.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_Min_bank">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>220</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-90.000000000000000</double>
   </property>
   <property name="maximum">
    <double>90.000000000000000</double>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Get_limit">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>460</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Загрузить лимиты</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Set_limit">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>460</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Отгрузить лимиты</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Get_trim">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>460</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Загрузить тримера</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Set_trim">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>460</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Отгрузить тримера</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_23_info_for_user">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>130</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Угол магнитного склонения:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label__24_info_for_user">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>160</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Тримирования оси тонгажа:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_25_info_for_user">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>190</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Тримирование оси крена:</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_Magnetc_declination">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>130</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <double>0.000000000000000</double>
   </property>
   <property name="maximum">
    <double>360.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_offset_pitch_axis">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>160</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-10.000000000000000</double>
   </property>
   <property name="maximum">
    <double>10.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_offset_roll_axis">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>190</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-10.000000000000000</double>
   </property>
   <property name="maximum">
    <double>10.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_26_info_for_user">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>250</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Максимальная ошибка по тонгажу:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label__27_info_for_user">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>280</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Минмальная ошибка по тонгажу:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label__28_info_for_user">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>310</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Максимальная ошибка по крену:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label__29_info_for_user">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>340</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Минимальная ошибка по крену:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label__30_info_for_user">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>370</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Максимальная ошибка по рысканью:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_31_info_for_user">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>400</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Минимальная ошибка по рысканью:</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_Max_pitch_error_pid">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>250</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-90.000000000000000</double>
   </property>
   <property name="maximum">
    <double>90.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_Min_pitch_error_pid">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>280</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-90.000000000000000</double>
   </property>
   <property name="maximum">
    <double>90.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_Max_roll_error_pid">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>310</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-90.000000000000000</double>
   </property>
   <property name="maximum">
    <double>90.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_Min_roll_error_pid">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>340</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-90.000000000000000</double>
   </property>
   <property name="maximum">
    <double>90.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_Max_yaw_error_pid">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>370</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-90.000000000000000</double>
   </property>
   <property name="maximum">
    <double>90.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_Min_yaw_error_pid">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>400</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-90.000000000000000</double>
   </property>
   <property name="maximum">
    <double>90.000000000000000</double>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
